### ISSUE TITLE
Can't generate man pages for multi-project repositories

### ISSUE THREAD

**Issue by HoneyryderChuck** [sunaku/binman#5]
I'm targeting this use-case: multiple related projects under the same directory tree (like rails, where active* gems are organized in the same repo, but published separately). All the sub-projects are organized as a standard, rubygem, i.e. they contain a lib/ directory and a bin/ directory. There is a centralized Rakefile and Gemfile shared by all in the root directory. 

What I'd like to do: integrate binman in the separate publishing pipelines. For each project, I'd like to state the "target main directory", that is the directory from which to look for a bin/ directory, and where to generate the target man/ to. 

This library assumes the directory where the Rakefile is located to be the target directory. This works for the common case where one repo = one project, but not for this use case. My proposal: make this a customized parameter (?).

**Comment by sunaku** [comment]
Good question.  What happens when you run `binman-rake` inside one of your subdirectories?  I believe it should process the local `bin/` folder inside your subdirectory, therefore you might be able to accomplish your goal by visiting (in an automated fashion :robot:) each subdirectory and running `binman-rake` therein.

**Comment by HoneyryderChuck** [comment]
I've created my own workaround by defining my tasks and using the `Binman.text/html/roff` methods. I tried running some rake tasks in the sub directories (I don't remember if that one in particular), but it created the man/ in the root directory. I had the feeling that this was because the tasks were relying on the Rakefile's location, and there is only one in the root. 

Let me retest that tomorrow, and I'll come back at you with more results.
